src/main/kotlin/com/openlattice/shuttle/control/IntegrationStatus.kt

```diff
@@ -0,0 +1,8 @@
+package com.openlattice.shuttle.control
+
+enum class IntegrationStatus {
+    IN_PROGRESS,
+    SUCCEEDED,
+    FAILED,
+    QUEUED
```

Comment:
I propose we consider a `CANCELED` state as well tho super minor and not required at all

Reply:
what would you propose would signal the canceled status to be applied?